src/store/index.js

```diff
@@ -14,6 +14,8 @@ export default new Vuex.Store({
       state.projects[payload.project.id].sprints = []
     },
     removeProject(state, payload) {
+      console.log(state);
+      console.log(payload);
```

Comment:
Are you just adding the UI and not the actual functionality?

Reply:
Yeah, just the UI stuff for now. I'll wait till you get your current branch merged before adding it.